development-defaults.ini

```diff
@@ -68,21 +72,53 @@ attendee_badge = 3000, 29999
 one_day_badge = 30000, 39999
 
 
```

Comment:
Strictly speaking, if they really did want them as a separate database table, that would be easy to add in their custom ``anthrocon`` Sideboard plugin.  So they wouldn't need to change this in Uber itself per se.